eva-server/src/main/java/uk/ac/ebi/eva/server/ws/ArchiveWSServer.java

```diff
@@ -85,7 +81,7 @@ public QueryResponse countSpecies() {
 
     @RequestMapping(value = "/species/list", method = RequestMethod.GET)
     public QueryResponse getSpecies() {
-        return setQueryResponse(archiveEvaproDbAdaptor.getSpecies(properties.getProperty("eva.version"), true));
+        return setQueryResponse(archiveEvaproDbAdaptor.getSpecies(dbCollectionsProperties.getVersion(), true));
```

Comment:
See that the first parameter declared in the [ArchiveEvaproDbAdaptor#getSpecies](https://github.com/EBIvariation/eva-ws/blob/develop/eva-lib/src/main/java/uk/ac/ebi/eva/lib/metadata/ArchiveEvaproDBAdaptor.java#L96) signature is not used. Could you please remove that parameter from the interface and drop the `db.collections.version` property?